projects/igniteui-angular/src/lib/list/list.component.spec.ts

```diff
@@ -445,6 +446,129 @@ describe('List', () => {
         });
     }, 5000);
 
+    it('checking the panLeftTemplate is visible when left-panning a list item.', () => {
+        const fixture = TestBed.createComponent(ListWithPanningTemplatesComponent);
+        const list = fixture.componentInstance.list;
+        fixture.detectChanges();
+
+        const firstItem = list.items[0] as IgxListItemComponent;
+        const leftPanTmpl = firstItem.leftPanningTemplateElement;
+        const rightPanTmpl = firstItem.rightPanningTemplateElement;
+        const itemNativeElements = fixture.debugElement.queryAll(By.css('igx-list-item'));
+
+        /* Click and drag item left */
+        clickAndDrag(itemNativeElements[1], -0.3);
+        fixture.detectChanges();
```

Comment:
Remove all fixture.detectChanges() after clickAndDrag(), panItem() and tick() as they are not necessary.

Reply:
All occurrences removed as suggested.